conjur-api.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency 'activesupport'
```

Comment:
Now that `activesupport` 5.0 is out, we need to bump the `gem.required_ruby_version = '>= 2.2.2'`